### ISSUE TITLE
Cheat no longer works?

### ISSUE THREAD

**Issue by itzarty** [danielkrupinski/OsirisInventory#21]
Im not sure if this is only caused by my injector (ProcessHacker) or is it because the cheat no longer works with the current version of csgo

**Comment by AuG-24** [comment]
https://anonfiles.com/x9wcg2N9xf/OsirisInventory_dll

**Comment by AuG-24** [comment]
compiled from a working osiris base

**Comment by yxb6846928** [comment]
> 

Can you email me the latest "OsirisInventory" code?  

my mail：yxb6846928@gmail.com

**Comment by yxb6846928** [comment]
> Can you email me the latest "OsirisInventory" code?
> thanks~
> my mail：[yxb6846928@gmail.com](mailto:yxb6846928@gmail.com)



> compiled from a working osiris base

Can you email me the latest "OsirisInventory" code?
thanks~
my mail：[yxb6846928@gmail.com](mailto:yxb6846928@gmail.com)

**Comment by yxb6846928** [comment]
> compiled from a working osiris base

Can you email me the latest "OsirisInventory" code?
thanks~
my mail：[yxb6846928@gmail.com](mailto:yxb6846928@gmail.com)

**Comment by AuG-24** [comment]
I am sorry I cannot help you there. I deleted those files but kept only the dll. As a better alternative , I recently learnt how to disable gui features in osiris main cheat so it functions exactly like osiris inventory and without any options to give you an advantage in game. Would you like those files?

**Comment by yxb6846928** [comment]
Yes. I just need "inventory".


Because I am a rookie.

I don't know how to hide other functions of "osiris".


AuG-24 ***@***.***> 于2022年3月11日周五 13:52写道：

**Comment by yxb6846928** [comment]
Or you can tell me.
If annotated, where do I need to annotate?
Is it complicated?

xb y ***@***.***> 于2022年3月11日周五 14:14写道：

**Comment by AuG-24** [comment]
yeah very similar to annotating , i removed the gui of aimbot , backtrack , chams , sound , esp etc etc anything that gives you an advantage , i removed it .  Kept only inventory changer and some fps boosting features like remove fog , grass , shadows , post processing . I'll share the files soon but as of now i am sharing the tutorial and dll i compiled.

**Comment by AuG-24** [comment]
dll -> https://anonfiles.com/rfO5v3N8x7/OsirisInventory_1_dll

tutorial -> https://www.youtube.com/watch?v=3w3zSv_w-ug

**Comment by AuG-24** [comment]
will share the source files in 4 hours as i am at work XD

**Comment by AuG-24** [comment]
https://anonfiles.com/R0f8xfNdxc/Osiris-master_1_zip

**Comment by nopde** [comment]
> Im not sure if this is only caused by my injector (ProcessHacker) or is it because the cheat no longer works with the current version of csgo

It's not injecting or it's CRASHING?

**Comment by nopde** [comment]
Process Hacker doesn't have Valve Anti Cheat's anti inject bypass, maybe that's the issue.

**Comment by AuG-24** [comment]
> > Im not sure if this is only caused by my injector (ProcessHacker) or is it because the cheat no longer works with the current version of csgo
> 
> It's not injecting or it's CRASHING?

process hacker will give you instant ban if not used under -untrusted launch option . the cheat too got outdated , so i compiled a new one from osiris base .. u can check it out using csghost or something

**Comment by nopde** [comment]
> > > Im not sure if this is only caused by my injector (ProcessHacker) or is it because the cheat no longer works with the current version of csgo
> > 
> > 
> > It's not injecting or it's CRASHING?
> 
> process hacker will give you instant ban if not used under -untrusted launch option . the cheat too got outdated , so i compiled a new one from osiris base .. u can check it out using csghost or something

I know, but it doesn't gives "instant ban". It doesn't injects. I don't really know if the cheat is outdated.

**Comment by aibhan** [comment]
> 很抱歉我不能帮你。我删除了这些文件，但只保留了 dll。作为一个更好的选择，我最近学习了如何禁用 osiris 主作弊中的 gui 功能，使其功能与 osiris 库存完全相同，并且没有任何选项可以让您在游戏中获得优势。你想要那些文件吗？

Do you have contact information?

**Comment by yxb6846928** [comment]
> https://anonfiles.com/R0f8xfNdxc/Osiris-master_1_zip

This link hints that the file has been cancelled.

Can you repost the latest source code?

thanks

**Comment by yxb6846928** [comment]
I received the file,
thank you very much!

AuG-24 ***@***.***> 于2022年3月11日周五 19:47写道：
